src/testsuite/vec_f64_dummy.c

```diff
@@ -318,7 +317,7 @@ test_vf64_ibm128_vec (vf64_t lval)
 }
 
 #ifndef __clang__
-/* Disable for clange becuase does not support the asm constraints
+/* Disable for clang becuase does not support the asm constraints
    (%L) to access 2nd FRP of FPR pair. */
```

Comment:
nit, not in this patch, but s/FRP/FPR/
Also, this is an odd case, but should this asm contraint issue be handled in `configure`?

Reply:
These are required by the ABI for IBM long double so clang is out of spec here.

I will look at a configure test for later (not in this pull request).